uptane-standard.md

```diff
@@ -658,7 +658,7 @@ ECUs MUST have a secure source of time. An OEM/Uptane implementor MAY use any ex
 For an ECU to be capable of receiving Uptane-secured updates, it MUST have the following data provisioned at the time it is manufactured or installed in the vehicle:
 
 1. A sufficiently recent copy of required Uptane metadata at the time of manufacture or install. See the Uptane Deployment Considerations ({{DEPLOY}}) for more information.
-    * Partial verification ECUs MUST have the Root and Targets metadata from the Director repository.
+    * Partial verification ECUs MUST have the Targets metadata from the Director repository.
     * Full verification ECUs MUST have a complete set of metadata (Root, Targets, Snapshot, and Timestamp) from both repositories, as well as the repository mapping metadata ({{repo_mapping_meta}}).
```

Comment:
For full verification, you want the ECU to have the snapshot metadata because this prevents rollback / replay attacks.  You don't need all of the targets metadata.

For partial verification, I think it is better to have a version of targets to reduce the scope of rollback / replay of targets metadata.  This is specifically impactful for metadata that isn't directed specifically at the vehicle.

Reply:
Fair enough. Thanks for the explanation! Should we add that reasoning here or to somewhere in the deployment doc?